components/service/sync-autofill/src/test/java/mozilla/components/service/sync/autofill/GeckoCreditCardsAddressesStorageDelegateTest.kt

```diff
@@ -70,10 +73,32 @@ class GeckoCreditCardsAddressesStorageDelegateTest {
     }
 
     @Test
-    fun `onCreditCardFetch`() {
+    fun `GIVEN autofill enabled WHEN onCreditCardsFetch is called THEN it returns all stored cards`() {
         scope.launch {
-            delegate.onCreditCardsFetch()
+            val storage: AutofillCreditCardsAddressesStorage = mock()
+            val storedCards = listOf<CreditCard>(mock())
+            Mockito.doReturn(storedCards).`when`(storage).getAllCreditCards()
```

Comment:
Prehaps import `doReturn` 

```suggestion
            doReturn(storedCards).`when`(storage).getAllCreditCards()
```

Reply:
Sure. Thank you!